src/SFA.DAS.RoatpOversight.Domain/AppealsOversightReviews.cs

```diff
@@ -0,0 +1,28 @@
+﻿using SFA.DAS.ApplyService.Types;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public class AppealsOversightReviews
```

Comment:
this feels redundant as per previous comment

Reply:
Nah it fits with the existing pattern but has incorrect name. Should be something like **CompletedAppealOutcomes**

Reply:
agree

Reply:
fine